app/invocation/invocation_input_file_node.tsx

```diff
@@ -0,0 +1,57 @@
+import React from "react";
```

Comment:
nit: consider renaming this file `invocation_action_input_node.tsx` (remove "file" since it can also represent a dir, and add "action" to be consistent with `invocation_action_card`)